LUFA/Drivers/USB/Class/Common/HIDClassCommon.h

```diff
@@ -330,6 +330,23 @@
 		#define HID_KEYBOARD_SC_RIGHT_SHIFT                       0xE5
 		#define HID_KEYBOARD_SC_RIGHT_ALT                         0xE6
 		#define HID_KEYBOARD_SC_RIGHT_GUI                         0xE7
+		#define HID_KEYBOARD_SC_MEDIA_PLAY                        0xE8
```

Comment:
@eltang You are right that some of these macros shouldn't belong here. Back in the days I was clueless but now I know better.

According to [USB HID usage tables](http://www.usb.org/developers/hidpage/Hut1_12v2.pdf) the relevant pages are:

* Keyboard / Keymap Page (0x07), starting on page 53
* Consumer Page (0x0C), starting on page 75

The `HID_KEYBOARD_SC_*` macros should cover Keyboard / Keymap Page. It'd be worth to introduce another namespace like `HID_KEYBOARD_MEDIA_*` to cover the consumer page.

Does it seem right, @abcminiuser?

I'm about to implement this in the [UHK firmware](https://github.com/UltimateHackingKeyboard/firmware) and backport it to LUFA. No ETA on it yet but I plan to do it within several weeks.